support/src/main/java/org/jenkinsci/plugins/workflow/support/storage/SimpleXStreamFlowNodeStorage.java

```diff
@@ -151,7 +150,6 @@ private Tag(@Nonnull FlowNode node, @Nonnull List<Action> actions) {
     private static final Field FlowNode$parentIds;
 
     static {
-        XSTREAM.addCompatibilityAlias("com.cloudbees.workflow.cps.checkpoint.NotExecutedNodeAction", NotExecutedNodeAction.class);
```

Comment:
The agreement was that this alias could be registered in the plugin from whence `NotExecutedNodeAction` came, rather than here.